app/main.py

```diff
@@ -3,17 +3,16 @@
 
 from fastapi import FastAPI
 
-from services.database.database import init_db, AsyncSessionLocal
 from app.routers.router import setup_router as setup_router_v2
+from app.services.database.database import AsyncSessionLocal, init_db
```

Comment:
I only changed this file because the import for `services` could not be found, so I updated the import path to `app.services`.